CS342ApplicationUITests/InstructorsUITests.swift

```diff
@@ -26,5 +26,8 @@ class InstructorsUITests: XCTestCase {
         
         app.collectionViews.buttons["Vishnu Ravi, MD"].tap()
         XCTAssert(app.staticTexts["Vishnu 👋"].waitForExistence(timeout: 0.5))
+        
+        app.collectionViews.buttons["Brooke Krajancich"].tap()
```

Comment:
@BKrajancich  Really like that you have started submitting the assignment 1 early, the view already looks great. I have seen that you currently experience a build error in the GitHub Actions that results from a failing test case. You can also reproduce this locally if you run the `InstructorUITests`. The problem is in this line:

The UI test navigates into the detail view of Vishnu and is then stuck here. You can simply navigate back out there by calling:
```suggestion
        app.navigationBars.firstMatch.buttons["CS342"].tap()
        
        app.collectionViews.buttons["Brooke Krajancich"].tap()
```
before your test call. 

I would suggest that you move your test into your separate `MemberViews` subfolder test and execute it there (then you don't have to navigate back out of Vishnu's view before that) or add the line I have state above before you try to open your subview.